components/explorer/src/lib.rs

```diff
@@ -372,6 +372,10 @@ impl<'a> IntoIterator for &'a BlockWithTransactions {
 /// | `location` | [`TxLocation`] | Location of the transaction in the block |
 /// | `location_proof` | [`ListProof`]`<`[`Hash`]`>` | Proof of transaction inclusion into a block |
 /// | `status` | (custom; see below) | Execution status |
+/// | `time` | [`DateTime`]`<`[`Utc`]`>` | Commitment time* |
```

Comment:
I think it's better to have an empty line between table and text.

```suggestion
/// | `time` | [`DateTime`]`<`[`Utc`]`>` | Commitment time* |
///
```